consensus/replay.go

```diff
@@ -324,10 +324,12 @@ func (h *Handshaker) ReplayBlocks(
 			}
 
 			if res.ConsensusParams != nil {
+				// TimeIotaMs is not exposed to the application.
+				// Make sure we preserve it.
 				timeIotaMs := state.ConsensusParams.Block.TimeIotaMs
-				state.ConsensusParams = types.PB2TM.ConsensusParams(res.ConsensusParams)
-				// TimeIotaMs is not a part of ABCI consensus params but
-				// of the actual state. Make sure it does not get overwritten.
+				{
+					state.ConsensusParams = types.PB2TM.ConsensusParams(res.ConsensusParams)
+				}
```

Comment:
Why the extra `{}`-brackets? The new scope doesn't introduce / separate anything.

Reply:
![Screenshot 2019-03-11 at 22 05 33](https://user-images.githubusercontent.com/1282182/54146656-e3672580-4449-11e9-9207-95ed7e51f74e.png)

It separates reading from state.ConsensusParams and rewriting it with ABCI cs params